packages/landing/src/ui.ts

```diff
@@ -161,7 +161,7 @@ Your Service/App URL:
             this.projectionNztm.classList.remove('lui-button-active');
             this.apiXyz.classList.remove('display-none');
         }
-        const cfg: MapOptions = { ...this.basemaps.config, tileMatrix: TileMatrixSets.get(projection) };
+        const cfg: MapOptions = { ...this.basemaps.config, tileMatrix: tileMatrix };
```

Comment:
NIT: you don't need the `: tileMatrix`

Reply:
I see, `this.basemaps.config` have tileMatrix already.

Reply:
@blacha. Wait, I think need it, it used to prepare the url.